spec/i18n_routable/i18n_routable_spec.rb

```diff
@@ -160,44 +160,44 @@ def resolve(path, http_verb=:get)
 
   end
 
-  context 'hash_for helpers' do
-
-    it 'should return a localized route when using hash_for helpers passing in locale' do
-      I18n.locale = 'fr'
-      route_hash = hash_for_posts_url(:locale => 'es')
-      route_hash.should eql :locale=>'es', :action=>"index", :controller=>"posts", :use_route=>"posts", :only_path=>false
-      url_for(route_hash).should eql "http://www.example.com/es/puestos"
-
-      route_hash = hash_for_posts_url(:locale => :es)
-      route_hash.should eql :locale=>:es, :action=>"index", :controller=>"posts", :use_route=>"posts", :only_path=>false
-      url_for(route_hash).should eql "http://www.example.com/es/puestos"
-    end
-
-    it 'should respect I18n.locale' do
-      route_hash = hash_for_post_url(:id => 1)
-      route_hash.should eql :id=>1, :action=>"show", :controller=>"posts", :use_route=>"post", :only_path=>false
-      url_for(route_hash).should eql "http://www.example.com/posts/1"
-
-      I18n.locale = 'es'
-
-      route_hash = hash_for_post_url(:id => 1)
-      route_hash.should eql :id=>1, :action=>"show", :controller=>"posts", :use_route=>"post", :only_path=>false
-      url_for(route_hash).should eql "http://www.example.com/es/puestos/1"
-    end
-
-  end
-
-  context 'url_for' do
-
-    it 'should recognize non named routes' do
-      I18n.locale = 'es'
-
-      route_hash = hash_for_post_url(:id => 1)
-      route_hash = route_hash.slice!(:use_route)
-      route_hash.should eql :id => 1, :action => "show", :controller => "posts", :only_path => false
-      url_for(route_hash).should eql "http://www.example.com/es/puestos/1"
-
-    end
-  end
+  # context 'hash_for helpers' do
```

Comment:
Yes. It’s a shame that hash_for_* has been deprecated. But you can always call recognize_route on url_for_*, though that seems a bit back-assward if you ask me.